index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+
+<body>
+    <main class="top-section">
+        <section class="header">
+            <div class="app-bar">
+                <ul>
+                    <li class="app-bar-item1"><img src="/images/icons/time.png" alt="Time"></li>
+                    <li class="app-bar-item2"><img src="/images/icons/Group4.png" alt="Phone Status"></li>
+                </ul>
+            </div>
+            <nav class="menu-icon">
+                <ul>
+                    <li class="menu-item1"><img src="/images/icons/logo.png" alt="Logo"></li>
+                    <li class="menu-item2"><img src="/images/icons/Union.png" alt="Main menu"></li>
+                </ul>
+            </nav>
+        </section>
```

Comment:
I'd suggest you make the following changes to this part of your code ;

- [ ] Please the first section in the `body` of your portfolio is the `header` tag,to show the heading of a webpage containing the logo and navigation. That is, `line 15-28` should be moved to start from`line 14` and the `section` tag should be replaced with a `header` tag 😄  

- [ ] The time and battery bars you included is not a part of the main design and should not be included in your portfolio, as it just shows the mobile time and battery status when the design was taken . Therefore, I'd suggest you kindly remove `lines 16 - line 21` from your code 👍 
- [ ] The logo icon on `line 24`   should be in a `a href` tag to show good user semantics and convention